starter-pack.rst

```diff
@@ -1,12 +1,11 @@
 .. _starter-pack:
 
-==============================
 The documentation starter pack
 ==============================
 
-The `starter pack repository <https://github.com/canonical/starter-pack>`_ provides a setup for a documentation repository that uses a Diátaxis structure and a Canonical-like theme.
+The `starter pack repository`_ provides a setup for a documentation repository that uses a Diátaxis structure and a Canonical-like theme.
 
-See the `Canonical starter pack <https://canonical-starter-pack.readthedocs-hosted.com/>`_ on Read the Docs for the output.
+See the `Canonical starter pack`_ on Read the Docs for the output.
 
 You can use this starter pack as a basis for your own documentation.
-See the `repository ReadMe <https://github.com/canonical/starter-pack/blob/main/readme.rst>`_ for setup instructions.
+See the `starter pack repository ReadMe`_ for setup instructions.
```

Comment:
Should we call it by its new name of `sphinx-docs-starter-pack`?

Reply:
Nice, didn't see it was renamed already. :)
I changed it for the main link above, but here I think just "starter pack" is sufficient.